admin/acceptance.py

```diff
@@ -365,7 +375,11 @@ def main(args, base_path, top_level):
 
     try:
         nodes = runner.start_nodes()
-        result = run_tests(nodes, options['trial-args'])
+        configure_cluster(control_node=nodes[0], agent_nodes=nodes)
+        result = run_tests(
+            nodes=nodes,
+            control_node=nodes[0], agent_nodes=nodes,
```

Comment:
It isn't necessary. For the tests, it doesn't seem to make a lot of sense to spin up an extra node for it, at least at this point.